modules/reactor/src/meshgenerators/PolygonConcentricCircleMeshGeneratorBase.C

```diff
@@ -928,5 +948,112 @@ PolygonConcentricCircleMeshGeneratorBase::generate()
   if (flat_side_up)
     MeshTools::Modification::rotate(*mesh0, 180.0 / (Real)_num_sides, 0.0, 0.0);
   mesh0->set_isnt_prepared();
+
+  if (_ring_block_ids.size() > 2 && getParam<bool>("replace_inner_ring_with_delaunay_mesh"))
+  {
+    // remove elements of the inner ring and add the boundary side set
```

Comment:
are there are any EEIDs that need to be copied over?
```suggestion
    // remove elements of the inner ring and create an outer-pin boundary side set
```

Reply:
Possibly yes. We may have ring id and sector id. I will add the code for this.

Reply:
Added ring_id and sector_id when they are requested.